src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -1,43 +1,101 @@
 // @flow
 import * as React from 'react';
-import { Image, StyleSheet, View } from 'react-native';
+import { Image, PanResponder, StyleSheet, View } from 'react-native';
+import { type PressEvent } from 'react-native/Libraries/Types/CoreEventTypes';
+import type {
+    GestureState,
+    PanResponderInstance,
+} from 'react-native/Libraries/Interaction/PanResponder';
 import { Path, Shape, Surface } from '@react-native-community/art';
 import tilebelt from '@mapbox/tilebelt';
 import type {
     BBOX,
     Point,
     Polygon,
-    ProjectType,
+    SingleImageryProjectType,
```

Comment:
where is this `SingleImageryProjectType` defined? I see it for the first time. What is it actually referring to? "normal" build area projects?

Reply:
It's defined in `flow-types.js`. Basically, it means "not Change Detection" I think, so projects that only use a single source of imagery. The way it is defined now, Completeness projects would be single imagery, which is weird. We should update the definitions for those.

Reply:
#435